scripts/ci/workflows-issues-manager/issues.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { ISSUES_MANAGER_TAG } = require("./constants.js");
+
+const getRepoIssues = async ({ github, context }) => {
+  let issues = [];
+
+  //TODO: Use a label or an id to get more specific issues
+  for await (const response of github.paginate.iterator(github.rest.issues.listForRepo, {
+    owner: context.repo.owner,
+    repo: context.repo.repo,
+    per_page: 100,
+  })) {
+    issues = issues.concat(response.data);
+  }
+
+  return issues;
+};
```

Comment:
@Abacn could you create a label called `issues-manager`? The issues created by this workflow will have that label. Could be better if we get the issues already filtered from the API instead of getting the ~4K issues.

Reply:
created: https://github.com/apache/beam/labels/issues-manager